### DISCUSSION TITLE
Is it possible to clear exif data?

### DISCUSSION THREAD

**Discussion by Arctomachine** [vladmandic/sdnext#2894] [Q&A]
Is there built in functionality to clear exif for selected image? And if not, can you add this feature?

**Comment by vladmandic** [comment]
You have option not to save image metadata.
Editing existing image just to remove metadata is trivial, but not something that should be inside the app.

**Comment by brknsoul** [comment]
https://chat.openai.com/share/48338969-f9b9-4ad8-bff8-3394df0a2586

GUI option follows Command Line option.

**Comment by kalle07** [comment]
it give only JPG save option?
beacuse the metadata can only be copied in 3 steps in ifanview (exif-data)
in PNGs (like automatic1111) you open notepad editor and you can see all information in the first view lines and i can copy it easy.
